devicetypes/timevalve-gaslock-t-08/timevalve-smart.src/timevalve-smart.groovy

```diff
@@ -0,0 +1,281 @@
+metadata {
+        definition (name: "Timevalve Smart", namespace: "timevalve.gaslock.t-08", author: "ruinnel") {
+                capability "Valve"
```

Comment:
Note: If the name of the smartapp changes then the file name needs to change as well, which last I heard doesn't update. This may require a new SmartApp creation.